migrate2rocky/migrate2rocky.sh

```diff
@@ -144,23 +145,23 @@ exit_message() {
 
 final_message() {
     errmsg '%s ' \
-	"An error occurred while we were attempting to convert your system to" \
-	"Rocky Linux. Your system may be unstable. Script will now exit to" \
-	"prevent possible damage."$'\n\n'
+        "An error occurred while we were attempting to convert your system to" \
+        "Rocky Linux. Your system may be unstable. Script will now exit to" \
```

Comment:
missing blank at the beginning of the line?
\" Rocky Linux. Your system may be unstable. Script will now exit to...
(or at the end of the previous)

Reply:
Blank is in the format specifier '%s '